book/accessibility.md

```diff
@@ -616,7 +621,8 @@ class CSSParser:
         assert self.word() == "media"
         self.whitespace()
         self.literal("(")
-        (prop, val) = self.pair()
+        self.whitespace()
```

Comment:
I believe so—we weren't allowing constructs like `@media( asdf: blah) { ... }`, note the space after the parenthesis.

I would add... The type of parser we're using here is a scannerless LL parser, where scannerless means there's no separate lexing/tokenization phase. Whitespace is exactly why typically people _don't_ write scannerless parsers! So this is a classic bug. In the book we use scannerless because it's typically less code and we're not a compiler book and can take some shortcuts, so with this bug I think we accept that we have no guards against it and are just gonna fix it where it crops up. In general the rule is you've gotta call `whitespace` between any other kinds of things.

Reply:
Added a test, which uncovered more parsing issues, which I've now also fixed.